parsers/US_NEISO.py

```diff
@@ -30,7 +38,7 @@
 }
 
 
-def timestring_converter(time_string):
+def timestring_converter(time_string: str) -> datetime:
     """Converts ISO-8601 time strings in neiso data into aware datetime objects."""
 
     dt_naive = arrow.get(time_string)
```

Comment:
If they are proper ISO-8601 datetimes I think both me an Robin would prefer to use datetime.fromisoformat() instead of manually assigning the TZ info for the datetime. If the string is missing the timezone I think we would still prefer using datetime.fromisoformat() but chaining on astimezone() as well. That way we can avoid using arrow all together for this.